ota-plus-web/conf/application.conf

```diff
@@ -85,11 +85,11 @@ websrc {
   wp = "https://*.wp.com"
 
   default = "default-src 'self'"
-  script = "script-src 'self' 'unsafe-eval' "${websrc.auth0src}
+  script = "script-src 'self' 'unsafe-eval' " ${websrc.auth0src}" "${?dev.scriptSrc}
```

Comment:
Could you elaborate on where dev.scriptSrc is coming from?
Will this affect only dev/prod or both? If former is the case when how do we distinguish between dev and prod in this case to make sure we won't allow arbitrary js loading convenience to leak into production?

Reply:
It should be set manually in deployment on k8s cluster.

Reply:
I think it's worth at least adding comment about it to make sure we won't accidentially screw this up later on - things like "unsafe-eval" always give me creeps :)

Reply:
There is no place in the config defining this value, so it is fine IMO